### PR TITLE
Ssh passkey

### PR DESCRIPTION
**Pull request by ssatish**
Added a new ssh key with passphrase

### CODE REVIEW

File: modules/users/manifests/seemasatish.pp [hunk 1]
```diff
@@ -3,6 +3,6 @@
   govuk::user { 'seemasatish':
     fullname => 'Seema Satish',
     email    => 'seema.satish@digital.cabinet-office.gov.uk',
-    ssh_key  => 'ssh-rsa REPLACE ME',
+    ssh_key  => 'ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCf+ssK23UspIqMTy3Fojqom1YQpaaPwtK/kmZMpuUBux0AGM+hfSkwYUMYHQzjZZ8sYQJLEEj+i4xi6nF7V2S9bl4SE1zClnPP8cAgrXczLjLraiv6amyDFDvJrrywN8M441Xvp0JubgRFyDAYRxZzT3SX+a9KwNl87sLKvmRJ+zqi5guJqD4onherTcrPpGqWNv17+WtkPzESg7FrG44nXo7Qg5+u/E+jTOqS+YIHjS9JKCmFkubR0iNPPU2axcLebGs9c3eayF7TepBtVrTVj+Qc7eubA+xvLLpXuYZr69cYzyUD/nMejH62L/EM0jenqX8EAB94iQNeDvainVFT ssatish@eussatish.local,
```
Comment (alexmuller) [inline modules/users/manifests/seemasatish.pp]: I think there's a closing quote `'` missing from this line.

**Comment by ssatish** [comment]
Added the missing quote

**Comment by mattbostock** [comment]
Hi @ssatish, I notice that this is a 2048-bit key. That's sufficient, but using 4096-bits would allow for a better a security margin. Would you mind recreating your key? You can use the command below to create a 4096-bit key.

```
ssh-keygen -t rsa -b 4096
```

**Comment by ssatish** [comment]
@mattbostock Have added 4096 bits key

**Pull request merged**